### PR TITLE
feat($NavBar): make navbar responsive

### PR DESCRIPTION
**Pull request by matejoslav**

### CODE REVIEW

File: web-app/src/components/Navbar.js [hunk 1]
```diff
@@ -32,6 +32,33 @@ const Navbar = () => {
...
+    background: white;
+    transform: ${({ open }) => (open ? "translateX(0)" : "translateX(-100%)")};
+    height: 100vh;
+    text-align: left;
+    padding: 2rem;
+    position: absolute;
+    top: 0;
+    left: 0;
+    transition: transform 0.3s ease-in-out;
+
```
Comment (nicholasmorgan) [inline web-app/src/components/Navbar.js:51]: Position needs to be set to fixed. In mobile, when you open the Navbar, you can scroll down past it.

Comment (matejoslav) [reply web-app/src/components/Navbar.js:51]: try again and let me know

**Review by MadelinTom — APPROVED** [review]
Looks great!

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201118-164239 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201118-164239") to [dev](https://console.bluemix.net/apps/1b7a0274-0655-4add-bc08-7f44a83df02d?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 5932cd3774fe035d1e5c6bff111a128b91e48b60

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201202-101709 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201202-101709") to [dev](https://console.bluemix.net/apps/56ba9f85-8947-4840-bdf0-ccef4ecd4149?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 5932cd3774fe035d1e5c6bff111a128b91e48b60

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20201208-234333 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20201208-234333") to [test](https://console.bluemix.net/apps/1e190213-4dab-4fe7-9972-03a4bc17b632?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:test"), including commit 5932cd3774fe035d1e5c6bff111a128b91e48b60

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201217-145052 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-dev-20201217-145052") to [dev](https://console.bluemix.net/apps/33545de7-22de-43a2-be66-d6f6744fc5b9?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:dev"), including commit 5932cd3774fe035d1e5c6bff111a128b91e48b60

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210205-152747 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-test-20210205-152747") to [test](https://console.bluemix.net/apps/1d38058e-0941-478d-ab73-1c5c09e8e146?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:test"), including commit 5932cd3774fe035d1e5c6bff111a128b91e48b60

**Comment by arnoldst** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/0a72695f-1fa5-4df5-9d98-aaa218c70126?env_id=ibm:yp:eu-de "lrp-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/66e6d48f-2aa6-41b2-a82d-e4fac4a78dc9 "lrp-toolchain") deployed [LRP-web-app](https://github.com/lebanon-relief-project/LRP-web-app/releases/tag/deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210303-110944 "deploy-ibm-yp-eu-de-Lebanon_Relief_Network-prod-20210303-110944") to [prod](https://console.bluemix.net/apps/f1598887-083c-4cac-a3a3-4950b5d061d5?env_id=ibm:yp:eu-de "ibm:yp:eu-de:Lebanon_Relief_Network:prod"), including commit 5932cd3774fe035d1e5c6bff111a128b91e48b60

**Pull request merged**